src/pages/FundingOpen/PasswordSetPage.jsx

```diff
@@ -162,83 +129,60 @@ export const SLocker = ({ color }) => {
 
 const SLayout = styled.div`
   display: flex;
-  flex-direction: column;
-  align-items: center;
-
-  width: 100%;
-
-  background-color: white;
+  flex-flow: column nowrap;
+  height: 100vh;
 `;
+
 const SForm = styled.form`
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  gap: 24px;
-
+  flex-flow: column nowrap;
   width: 335px;
-  margin: 24px 20px 0 20px;
+  margin: 20px 20px 0px 20px;
```

Comment:
컨벤션에 따라 `0`의 경우 `px` 등 단위를 붙이지 않고 `0`으로만 작성하는 것이 맞습니다!

```suggestion
  margin: 20px 20px 0 20px;
```